tensorflow/contrib/learn/python/learn/estimators/estimator_test.py

```diff
@@ -955,10 +955,11 @@ def test_export_savedmodel(self):
         self.assertTrue('input_example_tensor' in graph_ops)
         self.assertTrue('ParseExample/ParseExample' in graph_ops)
         self.assertTrue('linear/linear/feature/matmul' in graph_ops)
-        self.assertSameElements(
-            ['bogus_lookup', 'feature'],
-            graph.get_collection(
-                constants.COLLECTION_DEF_KEY_FOR_INPUT_FEATURE_KEYS))
+        self.assertItemsEqual(
+          ['bogus_lookup', 'feature'],
+          [x.decode("utf-8") for x in graph.get_collection(
```

Comment:
Does this work on Python 2.x with non-Unicode strings?

It might be better to use the `compat.py` conversion functions to canonicalize these to bytes (or str) objects....